### PR TITLE
Create a gallery

### PR DESCRIPTION
**Pull request by amitark7**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  "images/img6.jpeg",
+  "images/img7.jpeg",
+  "images/img8.jpg",
+  "images/img9.jpeg",
+  "images/img10.jpeg",
+  "images/img11.jpg",
+];
+
+//Get all HTML Element through map function
+const imagesElement = imagesArray.map(
```
Comment (amrit56789) [inline index.js]: change variable name createImageElement.

File: index.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+.container {
+    position: relative;
```
Comment (meanstack9716) [inline index.css]: add tailwind class for this

File: index.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    width: 100%;
```
Comment (meanstack9716) [inline index.css]: add tailwind class for this

File: index.css [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    height: 100vh;
```
Comment (meanstack9716) [inline index.css]: add tailwind class for this

File: index.css [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.container .backShadow {
+    width: 100%;
```
Comment (meanstack9716) [inline index.css]: add tailwind class for this

File: index.css [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    height: 100vh;
```
Comment (meanstack9716) [inline index.css]: add tailwind class for this

File: index.css [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    background-color: black;
```
Comment (meanstack9716) [inline index.css]: add tailwind class for this

File: index.css [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+    position: absolute;
```
Comment (meanstack9716) [inline index.css]: add tailwind class for this

File: index.css [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.container .popup-container {
+    width: 60%;
+    position: absolute;
```
Comment (meanstack9716) [inline index.css]: add tailwind class for this

File: index.css [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+
+.container .backShadow {
+    width: 100%;
+    height: 100vh;
+    background-color: black;
+    position: absolute;
+}
+
+.container .popup-container {
+    width: 60%;
```
Comment (meanstack9716) [inline index.css]: add tailwind class for this

File: index.css [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+    position: absolute;
+    left: 20%;
```
Comment (meanstack9716) [inline index.css]: add tailwind class for this

File: index.css [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+    top: 10%;
```
Comment (meanstack9716) [inline index.css]: add tailwind class for this

File: index.css [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.popup-container #popup-images {
+    width: 100%;
```
Comment (meanstack9716) [inline index.css]: add tailwind class for this

File: index.css [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+
+.buttons .btn {
+    position: absolute;
+    font-size: 20px;
+    top: 50%;
+    padding: 4px;
+    background-color: white;
+    color: black;
+    border: 2px solid white;
+    cursor: pointer;
```
Comment (meanstack9716) [inline index.css]: all these add tailwind class for this

File: index.html [hunk 15]
```diff
@@ -4,11 +4,36 @@
...
+    <div class="container mx-auto mt-2 h-screen relative 2xl:w-full">
+        <h1 class="text-3xl font-semibold tracking-wide text-center">Image Gallery</h1>
+        <div class="flex flex-wrap mt-2" id="image-container">
+        </div>
+        <div class="w-full h-screen absolute bg-black top-0 bg-opacity-70 hidden cursor-pointer pointer-events-auto"
+            id="back-drop"></div>
+        <div class="w-11/12 bg-white absolute top-36 left-4 border-8 hidden border-white pb-7 shadow-md overflow-hidden sm:w-10/12 sm:left-16 sm:pb-9 lg:w-8/12 lg:left-1/4 xl:w-6/12 xl:top-28"
+            id="popup-container">
+            <div class="w-full h-56 sm:h-72 md:h-80 xl:h-96">
+                <img src="https://caring-painddt.surge.sh/images/Images1.jpg" class="w-full h-full " alt="popupimages"
```
Comment (meanstack9716) [inline index.html]: use local image

**Review by meanstack9716 — CHANGES_REQUESTED** [review]

**Pull request merged**